src/content/docs/creating-a-project/creating-a-project.md

```diff
@@ -0,0 +1,16 @@
+## Creating a Project
+
+1. Sign In: Log in to your Angor account.
```

Comment:
We don't have login to the account and only the wallet is created or recovered

Reply:
Oh yeah I missed this out, will make adjustments.